Presentation/Sources/Presentation/ViewModel/HeadViewModel.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+import Domain
+
+class HeadViewModel: HeadViewModelProtocol {
+    
+    var heads: [HeadViewData]
```

Comment:
```suggestion
    let heads: [HeadViewData]
```

let's use always constants.

Reply:
changed